tests/performance/wp-content/mu-plugins/server-timing.php

```diff
@@ -25,6 +25,11 @@ static function () use ( $server_timing_values, $template_start ) {
 
 				$server_timing_values['total'] = $server_timing_values['before-template'] + $server_timing_values['template'];
 
+				// While values passed via Server-Timing are intended to be durations,
+				// any numeric value can actually be passed.
+				// This is a nice little trick as it allows to easily get this information in JS.
```

Comment:
```suggestion
				/*
				 * While values passed via Server-Timing are intended to be durations,
				 * any numeric value can actually be passed.
				 * This is a nice little trick as it allows to easily get this information in JS.
				 */
```

Use multiline comment.

Reply:
JS doesn't follow the exact same coding standards as PHP; both conventions are acceptable here.

Reply:
@Mamaduka Even in JS it's not really _accepted_ AFAIK, see https://developer.wordpress.org/coding-standards/inline-documentation-standards/javascript/#multi-line-comments and https://make.wordpress.org/core/2022/11/24/proposal-amend-the-inline-documentation-standards-for-multi-line-comments/. There was another proposal 5 years ago at https://make.wordpress.org/core/2018/12/18/proposed-revisions-to-javascript-coding-standards/ but it didn't result in any changes to the handbook. That said, it's still common in Gutenberg 🤷